src/lib/holocene/textarea.svelte

```diff
@@ -5,6 +5,7 @@
   export let isValid: boolean = true;
   export let placeholder: string = '';
   export let rows: number = 5;
+  export let spellcheck: boolean = null;
```

Comment:
For my own knowledge: why `null` here instead of just unset or false?

Reply:
> All other attributes are included unless their value is [nullish](https://developer.mozilla.org/en-US/docs/Glossary/Nullish) (null or undefined).

I set it to `null` thinking that `spellcheck` should just not be included unless it is defined ([doc link](https://svelte.dev/docs#template-syntax-attributes-and-props))

Reply:
Nice ok that makes sense!